src/findRepl.tsx

```diff
@@ -50,30 +50,35 @@ export default function Command() {
       ) : null}
       <List.Section title="Your Repls" subtitle={data?.length + ""}>
         {data?.map((searchResult) => (
-          <SearchListItem key={searchResult.title} searchResult={searchResult} />
+          <SearchListItem key={searchResult.title} searchResult={searchResult} currentUserId={userId} />
         ))}
       </List.Section>
     </List>
   );
 }
 
-function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
-  console.log(searchResult);
+function SearchListItem({ searchResult, currentUserId }: { searchResult: SearchResult; currentUserId: number | null }) {
+  let title = searchResult.title;
+
+  if (currentUserId !== searchResult.owner.id) {
+    title = `@${searchResult.owner.username}/${title}`;
+  }
+
   return (
     <List.Item
-      title={searchResult.title}
+      title={title}
       subtitle={searchResult.description}
       icon={{ source: searchResult.iconUrl, mask: Image.Mask.RoundedRectangle }}
       accessories={[
```

Comment:
We could maybe add the author / owner as a second accessory!

Reply:
good idea, this is what it looks like:

<img width="763" alt="Screenshot 2023-06-01 at 15 53 54" src="https://github.com/devinhalladay/replit-raycast/assets/6189367/fb328b49-96e0-4d2a-bb85-8416a20662d4">